content/introduction/use-cases/data-and-model-quality.md

```diff
@@ -28,6 +28,16 @@ Dolt is the only database with [branch](../../concepts/dolt/git/branch.md) and [
 
 Lastly, [commits](../../concepts/dolt/git/commits.md), [logs](../../concepts/dolt/git/log.md), and [diffs](../../concepts/dolt/git/diff.md) can be used for model insights. Did Thursday's model perform better than Tuesday's but had the same model weights? Inspect the data diff to see what changed. Inspect the commit log to see where that new data came from.
 
+# Dolt replaces...
+
+## Unstructured files in cloud storage
+
+It is common practice to store copies of training data or database backups in cloud storage for model reproducibility. A full copy of the data is stored for every training run. This can become quite expensive and limit the amount of models you can reproduce. Dolt stores only the differences between stored versions decreasing the cost of data storage. Additionally, Dolt can produce diffs between versions of training data producing novel model insights.
+
+## MySQL, Postgres, or other databases
+
+Dolt can replace any database used to store and query data. Many of our customers switch from other OLTP databases like MySQL or Postgres to improve data and model quality through versioning. Customers have also switched to Dolt from document databases like MongoDB. Dolt's additional unique features like branches, diffs, and merges allow for human review of data changes and multiple parallel data projects.
```

Comment:
A similar but maybe slightly different observation from Turbine, software processes whose result is data can be difficult to inspect as a human. As an example, a 100 SQL query DBT pipeline can help break up long analytics workflows into chains of smaller tasks, but as a human you still need diffs to understand and verify the results.

Reply:
This is a good insight. Not sure if it fits here though.